syntax = "proto3";

package users;

message UserInfo{
    string name = 1;
    string last_name = 2;
    string age = 3;
    string email = 4;    
    string phone = 5;
}

message Empty {
}

service User {
    rpc GetUser(Empty) returns (UserInfo);
}